batcher/aligned-batcher/src/metrics.rs

```diff
@@ -13,6 +13,7 @@ pub struct BatcherMetrics {
     pub reverted_batches: IntCounter,
     pub batcher_started: IntCounter,
     pub gas_price_used_on_latest_batch: IntGauge,
+    pub broken_ws_connections: IntGauge,
```

Comment:
This should be a counter, not a gauge

Reply:
Alternatevly if we want to count how many were resetted per batch (not the current implementation but maybe the intended functionality), we could use a [CounterVec](https://docs.rs/prometheus/latest/prometheus/type.CounterVec.html)

Reply:
I changed it to a `IntCounter`, we might consider `CounterVec` in the future.

Reply:
Actually, the counter is ok, since we do not have a measure against a reference anymore, we are just incrementing.